lib/staccato/adapter/http_poison.ex

```diff
@@ -1,8 +1,11 @@
 defmodule Staccato.Adapter.HttpPoison do
-  # @behaviour Staccato.Adapter
-
   def post(url, params) do
-    case HTTPoison.post url, URI.encode_www_form(params) do
+    encoded_params = URI.encode_query(params)
+    case url
+         |> HTTPoison.post(encoded_params, [
+           {"User-Agent", params["ua"]},
```

Comment:
Oh, I guess I wasn't very clear there. What did you need to set the User-Agent for? Is it being picked up by google analytics? I think the `ua` param will (hopefully) set the same value in GA.